### PR TITLE
Updated text in Degeneracy Hunter documentation.

### LINKED ISSUE #880: Change link to Degeneracy Hunter example
Current link is into a GitHub PR. Should be:  https://idaes.github.io/examples-pse/latest/Examples/Tools/degeneracy_hunter_doc.html

### PR DESCRIPTION
**Pull request by adowling2**
## Changes proposed in this PR:
- Edits to documentation (one file)

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by adowling2** [comment]
@dangunter @lbianchi-lbl @ksbeattie Quick review?

File: docs/explanations/modeling_extensions/diagnostics/index.rst [hunk 1]
```diff
@@ -2,9 +2,8 @@
...
 
-Degeneracy Hunter is a collection of tools for diagnostics of mathematical programs. The core ideas behind Degeneracy Hunter are explained here: https://www.sciencedirect.com/science/article/pii/B9780444635785501304
-
-Degeneracy Hunter is currently included in IDAES for beta testing purposes. This page will be updated as part of the official release. If you would like to help test Degeneracy Hunter, please look at the example notebook available here: https://github.com/IDAES/examples-pse/pull/21 Please report any thoughts, questions or bugs to: adowling@nd.edu
-
-Full documentation of the available methods can be :ref:`found here<reference_guides/core/util/model_diagnostics:Degeneracy Hunter>`.
+Degeneracy Hunter is a collection of tools for diagnostics of mathematical programs.
 
+* `Core ideas (conference paper) <https://www.sciencedirect.com/science/article/pii/B9780444635785501304>`_
+* `Example notebook <https://idaes.github.io/examples-pse/latest/Examples/Tools/degeneracy_hunter_doc.html>`_
```
Comment (andrewlee94) [inline docs/explanations/modeling_extensions/diagnostics/index.rst:8]: We will need to update this soon to point to the new examples repo, but other wise looks good.

@dangunter Any guidance on whether we should update the examples link now, or merge this as is and do another change later?

**Review by andrewlee94 — APPROVED** [review]

**Review by eslickj — APPROVED** [review]

**Comment by lbianchi-lbl** [comment]
I'll let @dangunter decide but I feel that, given that we'll have to update all links on https://idaes-pse.readthedocs.org to point to the new examples repository, we might as well merge this as-is and then change it later with the rest.

**Comment by adowling2** [comment]
@lbianchi-lbl I like your decisive action.

**Pull request merged**